app/javascript/components/ingredients/ingredients-container.vue

```diff
@@ -121,7 +121,7 @@ import { getIngredients, postIngredient, deleteIngredient, editIngredient } from
 import IngredientsForm from './ingredients-form';
 import IngredientsTable from './ingredients-table';
 import SearchMarketIngredients from './search-market-ingredients';
-
+import Vue from 'vue';
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  Expected 1 empty line after import statement not followed by another import  import/newline-after-import